app/controllers/api/v1/upvotes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::UpvotesController < ApplicationController
+  def create
+    upvote = Upvote.new(user: current_user)
+    upvote.review = Review.find(params[:review_id])
+    reviews = Review.where(videogame: upvote.review.videogame)
+    if upvote.save 
+      if current_user.downvotes.find_by(review_id: params["review_id"]) != nil
```

Comment:
this is allowable potentially DRY up this `if` statement with .present? or .any? 
https://www.ombulabs.com/blog/benchmark/performance/rails/present-vs-any-vs-exists.html